### PR TITLE
Feature/457 aggregate delay stats for journey

### PR DESCRIPTION
**Pull request by JaJako**
### Description

This PR lays the base to add further stats like about interchanges or cancellation in Backend. Mapping and Stats types are being changed in Backend. Basic adjustments to support new types in Frontend are made, too. First part of #457. 

## Checklist

- [x] My code follows the style guidelines
- [x] I have documented my code
- [x] My changes generate no new warnings
- [x] I have linked the Issue
- [x] The Pull Request covers all the Checkpoints from the tagged Issue
- [x] I have noted the hours of work i have put in the issue

## I have edited following Parts of the project

- [x] Frontend
- [x] Backend
- [ ] Github
- [ ] Root

## If tests are required did you make them?

- [ ] No tests required
- [x] Yes
- [ ] No

### CODE REVIEW

File: backend/src/historicData/service/historicDataQueryRunner.service.ts [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                                    FROM historic_with_delay
+                                    ORDER BY planned:: date, trip_code DESC)
+
+    SELECT MAX(delay), MIN(delay), AVG(delay), STDDEV(delay)
+    FROM latest_of_day_tripcode;
+`;
+
+export const INTERCHANGE_REACHABLE_QUERY = `
+
+`; // TODO
```
Comment (timvahlbrock) [inline backend/src/historicData/service/historicDataQueryRunner.service.ts:54]: I suspect these TODOS here are okay for now?

File: backend/src/historicData/service/historicData.service.ts [hunk 2]
```diff
@@ -1,31 +1,134 @@
...
+    maxDelay: parseFloat(result[0].max),
+    minDelay: parseFloat(result[0].min),
+    averageDelay: parseFloat(result[0].avg),
+    standardDeviation: parseFloat(result[0].stddev)
+  } as DelayStats;
 }
 
-export type DelayEntryWithEstimate = Required<DelayEntry>;
+interface InterchangeReachableQueryResult extends QueryResult {
+// TODO
```
Comment (timvahlbrock) [inline backend/src/historicData/service/historicData.service.ts:65]: TODO here is okay?

**Review by timvahlbrock — APPROVED** [review]

File: backend/src/historicData/service/historicDataQueryRunner.service.ts (hunk 1 above)
Comment (JaJako) [reply backend/src/historicData/service/historicDataQueryRunner.service.ts:54]: Yes, thought it is a good time to merge alle the changes so far (as base for additional stats) before working on the sql queries itself. (At the moment UnavaibleStats is getting returned for interchange reachable and cancellation stats. These TODOs should be gone in second part/PR of this issue.

File: backend/src/historicData/service/historicData.service.ts (hunk 2 above)
Comment (JaJako) [reply backend/src/historicData/service/historicData.service.ts:65]: see above

**Pull request merged**